invenio_communities/templates/semantic-ui/invenio_communities/details/macros/detail.html

```diff
@@ -0,0 +1,15 @@
+{#
+    Copyright (C) 2022 CERN.
+  
+    Invenio RDM Records is free software; you can redistribute it and/or modify
+    it under the terms of the MIT License; see LICENSE file for more details.
+  #}
+
+{% macro truncate_text(text) %}
+  {% if text|length > 200 %}
+    <p id="community-description-less">{{ text|truncate(200)|safe }} <button class="community-description-button">Show more.</button></p>
```

Comment:
`<p>` tags are usually reserved for text, and you are assigning logic to it, and a button. I would rather go with a `<div>` which contains both paragraph of text and a button. 
I think it also matters for accessibility reasons